### PR TITLE
Updated URL list according to new modules listed for production.

### PR DESCRIPTION
**Pull request by marcoooo**
New modules previously added were not picked up because not in the ``_default_modules`` function to get a URL.

### CODE REVIEW

**Comment by marcoooo** [comment]
Matthieu, in 'Files Changed' then Diff Settings > Ignore whitespace > reload :-) https://github.com/Ensembl/ensembl-git-tools/pull/31/commits/8347d9ac91c611f50f73bb657212880112d943d5?utf8=%E2%9C%93&diff=unified&w=1

File: bin/git-ensembl [hunk 1]
```diff
@@ -505,28 +507,28 @@ sub get_groups {
...
     $groups->{all} ||= {
-      desc => 'An auto-genreated group pointing at all modules known to the script. Use this to apply commands to all repositories',
-      modules => [keys %{$modules}],
+        desc    => 'An auto-generated group pointing at all modules known to the script. Use this to apply commands to all repositories',
+        modules => [ keys %{$modules} ],
     };
     $groups->{available} ||= {
-      desc => 'An auto-genreated group pointing at all modules known to the script, and available in the directory. Use this to apply commands to all available repositories',
-      modules => [grep -d, keys %{$modules}],
+        desc    => 'An auto-generated group pointing at all modules known to the script, and available in the directory. Use this to apply commands to all available repositories',
```
Comment (nerdstrike) [inline bin/git-ensembl]: Very good!

File: bin/git-ensembl [hunk 2]
```diff
@@ -570,134 +572,98 @@ sub _cmdline_cfg {
...
-    'eg-web-protists'     => $eg_root.'eg-web-protists.git',
-    'eg-web-g1k'          => $eg_root.'eg-web-g1k.git',
-    'eg-web-parasite'     => $eg_root.'eg-web-parasite.git',
-    'eg-web-testsuite'    => $eg_root.'eg-web-testsuite.git',
-    'eg-web-vectorbase'   => $eg_root.'eg-web-vectorbase.git',
-    'eg-rest'             => $eg_root.'eg-rest.git',
-    'ensemblgenomes-api'  => $eg_root.'ensemblgenomes-api.git',
-    'eg-web-ensembl-configs'  => $eg_root.'eg-web-ensembl-configs.git' 
+  my %def_modules;
+  my $def_groups = _default_groups($opts);
```
Comment (nerdstrike) [inline bin/git-ensembl]: I wish _default_groups were explicitly imported.

File: bin/git-ensembl [hunk 3]
```diff
@@ -570,134 +572,98 @@ sub _cmdline_cfg {
...
+  foreach my $outter (keys %{$def_groups}) {
+    foreach my $module (@{$def_groups->{$outter}->{modules}}) {
+      if ($module =~ m/^(eg-|ensemblgenomes-)/) {
+        $def_modules{$module} = $eg_root . $module . '.git';
```
Comment (nerdstrike) [inline bin/git-ensembl]: File::Spec::catfile would be safer to use here - no need to worry about forgetting trailing slashes.

File: bin/git-ensembl [hunk 4]
```diff
@@ -570,134 +572,98 @@ sub _cmdline_cfg {
...
-    'eg-web-g1k'          => $eg_root.'eg-web-g1k.git',
-    'eg-web-parasite'     => $eg_root.'eg-web-parasite.git',
-    'eg-web-testsuite'    => $eg_root.'eg-web-testsuite.git',
-    'eg-web-vectorbase'   => $eg_root.'eg-web-vectorbase.git',
-    'eg-rest'             => $eg_root.'eg-rest.git',
-    'ensemblgenomes-api'  => $eg_root.'ensemblgenomes-api.git',
-    'eg-web-ensembl-configs'  => $eg_root.'eg-web-ensembl-configs.git' 
+  my %def_modules;
+  my $def_groups = _default_groups($opts);
+  foreach my $outter (keys %{$def_groups}) {
```
Comment (nerdstrike) [inline bin/git-ensembl]: $outter should probably be something like '$group_name' for easier comprehension.

Comment (marcoooo) [reply bin/git-ensembl]: ok.

File: bin/git-ensembl (hunk 3 above)
Comment (marcoooo) [reply bin/git-ensembl]: Well, let's deal with this another time :-)

File: bin/git-ensembl (hunk 2 above)
Comment (marcoooo) [reply bin/git-ensembl]: This is the next function declared in the file, is there some kind of load order needed? (should I move the _default_groups declaration before?)

Comment (nerdstrike) [reply bin/git-ensembl]: Weird, I couldn't find it.

**Comment by nerdstrike** [comment]
I defer to @muffato on this.

File: bin/git-ensembl (hunk 1 above)
Comment (marcoooo) [reply bin/git-ensembl]: Thanks to my IDE :-)

**Comment by nerdstrike** [comment]
I've spent 8 years campaigning against whitespace shuffling and tabs. I'm sick of it.

**Comment by marcoooo** [comment]
Assuming the changes has been approved, are you ok to merge it?

**Pull request merged**